python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -257,60 +256,87 @@ def repackage_eimage(self, phosim_eimg_file, out_dir='.'):
             PhoSim eimage file.
         out_dir : str, optional
             Output directory (the default is '.'.)
+        verbose: bool, optional
+            Set to True to print out processing information.
         """
+        TELCODE = self.telcode
+        CONTRLLR = self.contrllr
 
         sensor = fits.open(phosim_eimg_file)[0]
 
-        # Set keywords in primary HDU, extracting most of the relevant
-        # ones from the original phosim eimage file.
+        filename = phosim_eimg_file
```

Comment:
Do you need this variable: `filename` to be assigned as `phosim_eimg_file` here?

Reply:
No, removed double-assignment